routes/user.router.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const UserService = require('../services/user.service');
+const { getUserSchema } = require('../schemas/user.schema');
+const validatorHandler = require('../middlewares/validator.handler');
+
+const router = express.Router();
+const service = new UserService();
+
+router.get(
+  '/:id',
+  validatorHandler(getUserSchema, 'params'),
+  async (req, res) => {
+    try {
+      const { id } = req.params;
+      const user = await service.findByPk(id);
+      if (!user) {
+        res.status(404).json({
+          message: 'Usuario no encontrado',
```

Comment:
This is the type of message we should put in the language JSON, the validation and JOI generated messages that are translated by the package